ios-trivia-game/Models/Answer.swift

```diff
@@ -31,6 +31,16 @@ class Answer: NSObject {
         self.timestamp = String(describing: NSDate())
     }
     
+    init(answerText: String) {
```

Comment:
why do you need this initializer?
since you need to populate these values, why not use the initializer above?

Remove this initializer and use the one above